ivadomed/utils.py

```diff
@@ -34,7 +34,7 @@ def __str__(self):
 
```

Comment:
The file `requirements.txt` has had its permissions changed. (You can see this in the ["Files changed"](https://github.com/ivadomed/ivadomed/pull/787/files) view of your PR.)

![Screenshot from 2021-07-02 11-04-08](https://user-images.githubusercontent.com/16181459/124294010-3edee500-db25-11eb-8427-a0b7d8a960ee.png)

Please modify this file so that it will have the same `100644` permissions as it had before:

```
chmod 644 requirements.txt
```

Then commit the changes.

Reply:
Done.